### PR TITLE
[Docs][Connector-V2][Hudi] Reconstruct the Hudi connector document

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
subtask of #4572 

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Carl-Zhou-CN** [comment]
@TyrantLucifer @Hisoka-X CC,Thanks

File: docs/en/connector-v2/source/Hudi.md [hunk 1]
```diff
@@ -2,69 +2,67 @@
 
 > Hudi source connector
 
-## Description
+## Support Those Engines
 
-Used to read data from Hudi. Currently, only supports hudi cow table and Snapshot Query with Batch Mode.
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/Hudi.md]: `SeaTunnel Zeta`

Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/source/Hudi.md]: done

**Review by Hisoka-X — DISMISSED** [review]

**Review by zhilinli123 — CHANGES_REQUESTED** [review]
change: `https://seatunnel.apache.org/docs/transform-v2/sql/`

File: docs/en/connector-v2/source/Hudi.md [hunk 2]
```diff
@@ -73,7 +71,15 @@ source {
     kerberos.principal = "test_user@xxx"
     kerberos.principal.file = "/home/test/test_user.keytab"
   }
+}
+
+transform {
+    # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+    # please go to https://seatunnel.apache.org/docs/transform/sql
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/Hudi.md]: change: `https://seatunnel.apache.org/docs/transform-v2/sql/`

Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/source/Hudi.md]: done

**Review by zhilinli123 — APPROVED** [review]

**Review by TaoZex — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**